packages/apps-config/src/api/spec/elysium.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017-2024 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { OverrideBundleDefinition } from '@polkadot/types/types';
+
+/* eslint-disable sort-keys */
+const definitions: OverrideBundleDefinition = {
```

Comment:
This file needs to be imported into `index.ts`, and exported as part of the `spec`.

Reply:
This file has been imported in index.ts.